test/tap/tests/pgsql-user-sync-unit-t.py

```
@@ -0,0 +1,11 @@
+#!/usr/bin/env python3
+import os
+import subprocess
+import sys
+from pathlib import Path
+
+root = Path(os.environ.get("WORKSPACE", Path(__file__).resolve().parents[3]))
+suite = root / "tools/pgsql_user_sync/tests/test_pgsql_user_sync.py"
+raise SystemExit(subprocess.call(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)